browser/ui/views/omnibox/brave_omnibox_view_views_browsertest.cc

```diff
@@ -73,6 +74,30 @@ IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, CopyCleanURLToClipboardTest) {
             "e=&f=g&=end");
 }
 
+IN_PROC_BROWSER_TEST_F(BraveOmniboxViewViewsTest, CopyURLToClipboardTest) {
+  brave::URLSanitizerServiceFactory::GetForBrowserContext(browser()->profile())
+      ->Initialize(R"([
+    { "include": [ "*://*/*"], "params": ["utm_content"] }
+  ])");
+  const std::string test_url(
+      "https://dev-pages.bravesoftware.com/clean-urls/"
+      "exempted/"
```

Comment:
It's a little weird to be using `exempted/` in these test cases because the JSON config we supply here does **not** in fact exempt this URL.

I'd suggest removing `/exempted/` in these three new test cases to reduce potential confusion.

Reply:
done